frontend/src/components/ViewActivities/activityfns.test.js

```diff
@@ -0,0 +1,95 @@
+import * as activityFns from './activityfns';
+
+// Date objects to be used in tests. All on May 2nd, 2020. 
+const ten = new Date(Date.UTC(2020,  4, 2, 10,  0));
+const eleven = new Date(Date.UTC(2020, 4, 2, 11, 0));
+const elevenThirty = new Date(Date.UTC(2020, 4, 2, 11, 30));
+const twelve = new Date(Date.UTC(2020, 4, 2, 12, 0));
+const one = new Date(Date.UTC(2020, 4, 2, 13, 0));
+
+// Date objects to be used in tests. All on different days.
+const may102pm = new Date(Date.UTC(2020, 4, 10, 14, 0));
+const may014pm = new Date(Date.UTC(2020, 4, 1, 16, 0));
+const may153am = new Date(Date.UTC(2020, 4, 15, 3, 0));
+
+function createActivity(startTime, endTime){
+  return {'start_time': startTime, 'end_time': endTime};
```

Comment:
change these to use db constants?

Reply:
+1
Using constants here (especially if they already exist) is preferred.